console_logging.go

```diff
@@ -88,13 +88,24 @@ var (
 	Color = false
 )
 
+func IsValid(file *os.File) bool {
```

Comment:
I don't think you are using this method in your PR

Fo you need it?

Reply:
it's used here: https://github.com/fortio/log/pull/66/files#diff-ff87b7c4777a35588053a509583d66c9f404ccbea9e1c71d2a5f224d7ad1323eR177

Reply:
Indeed